core/common/src/main/java/alluxio/util/CommonUtils.java

```diff
@@ -882,6 +885,12 @@ public static List<File> recursiveListLocalDir(File dir) {
    * see https://www.oracle.com/java/technologies/javase/versioning-naming.html for reference
    */
   public static int parseMajorVersion(String version) {
+    if (version.endsWith(EARLY_ACCESS_SUFFIX)) {
```

Comment:
it would be best to add these two scenarios in the unit test `dora/core/common/src/test/java/alluxio/util/CommonUtilsTest.java#parseVersion()`

Reply:
On it.

Reply:
Done. PTAL once again.

Reply:
Seems like tests are green